qiskit_ibm_runtime/base_primitive.py

```diff
@@ -24,15 +23,10 @@
 
 from qiskit.primitives.containers.estimator_pub import EstimatorPub
 from qiskit.primitives.containers.sampler_pub import SamplerPub
-from qiskit.providers.options import Options as TerraOptions
 from qiskit.providers.backend import BackendV1, BackendV2
 
-from .provider_session import get_cm_session as get_cm_provider_session
-
-from .options import Options
 from .options.options import BaseOptions, OptionsV2
-from .options.utils import merge_options, set_default_error_levels, merge_options_v2
-from .runtime_job import RuntimeJob
+from .options.utils import merge_options, merge_options_v2
```

Comment:
`merge_options` function still makes sense?

Reply:
I think we still need it for those that want to pass options as a `dict`